### PR TITLE
Add doc comments to all the account methods in nodejs bindings

### PR DESCRIPTION
**Pull request by oddgrd**
# Description of change

Added doc comments to all the methods on `Account` + a few nits and lints.

I managed to sneak in a breaking change, the `account.minimumRequiredStorageDeposit` erroneously took an array of outputs when it should only take a single `Output`.

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Documentation Fix

## How the change has been tested

I tested the `account.minimumRequiredStorageDeposit` in the nodejs example: `16b-prepare-output.js`.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: bindings/nodejs/lib/Account.ts [hunk 1]
```diff
@@ -123,13 +157,24 @@ export class Account {
         return JSON.parse(resp).payload;
     }
 
+    /**
+     * Generate a new unused address.
+     * @param options Options for address generation.
```
Comment (Thoralf-M) [inline bindings/nodejs/lib/Account.ts:162]: Options should be there only once?

File: bindings/nodejs/lib/Account.ts [hunk 2]
```diff
@@ -123,13 +157,24 @@ export class Account {
...
         options?: AddressGenerationOptions,
     ): Promise<Address> {
         const addresses = await this.generateAddresses(1, options);
         return addresses[0];
     }
 
+    /**
+     * Generate new unused addresses.
+     * @param amount The amount of addresses to generate.
+     * @param options Options for address generation.
```
Comment (Thoralf-M) [inline bindings/nodejs/lib/Account.ts:175]: Same here

**Review by Thoralf-M — DISMISSED** [review]
Thanks!

**Review by Thoralf-M — DISMISSED** [review]

**Review by samuel-rufi — DISMISSED** [review]

File: bindings/nodejs/lib/Account.ts [hunk 3]
```diff
@@ -123,13 +157,24 @@ export class Account {
...
+     * @returns The address.
```
Comment (Thoralf-M) [inline bindings/nodejs/lib/Account.ts]:
```suggestion
     * @returns The addresses.
```

**Review by Thoralf-M — DISMISSED** [review]

File: bindings/nodejs/lib/Account.ts [hunk 4]
```diff
@@ -162,6 +215,11 @@ export class Account {
         return JSON.parse(response).payload;
     }
 
+    /**
+     * Generate an the data of an output.
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/Account.ts]: Weird

File: bindings/nodejs/lib/Account.ts [hunk 5]
```diff
@@ -106,6 +132,14 @@ export class Account {
         return JSON.parse(resp).payload;
     }
 
+    /**
+     * Consolidates basic outputs with only an `AddressUnlockCondition` from an account
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/Account.ts]:
```suggestion
     * Consolidate basic outputs with only an `AddressUnlockCondition` from an account
```
Consistency

File: bindings/nodejs/lib/Account.ts [hunk 6]
```diff
@@ -355,23 +479,37 @@ export class Account {
         return JSON.parse(response).payload;
     }
 
+    /**
+     * Prepare a send amount transaction, useful for offline signing.
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/Account.ts:483]:
```suggestion
     * Prepare and send amount transaction, useful for offline signing.
```

File: bindings/nodejs/lib/Account.ts [hunk 7]
```diff
@@ -518,6 +701,12 @@ export class Account {
         return JSON.parse(response).payload;
     }
 
+    /**
+     * Syncs the account by fetching new information from the nodes.
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/Account.ts]:
```suggestion
     * Sync the account by fetching new information from the nodes.
```
Consistency

**Review by thibault-martinez — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**